doc/module-docs/cc_package_update_upgrade_install/example2.yaml

```diff
@@ -0,0 +1,8 @@
+#cloud-config
```

Comment:
Nit: It seems a bit odd to have 2 of our 3 examples dedicated to a workaround for a single package manager  on a single OS. I don't think we really need an example showing how to unhold it.

Reply:
sounds good to me. I'll drop that example. of needed it is a corner case for sure.